### DISCUSSION TITLE
"A" Axis limit switch

### DISCUSSION THREAD

**Discussion by Tob3ster97** [Jay-Tech/ioSenderTouch#13] [Q&A]
Is there an option to display the A axis limit swith at the top? I currently only have XYZ

**Comment by Jay-Tech** [comment]
Are you set as Portrait or landscape ? I was pretty sure I added 4th axis support back in.  As a quick check it shows A axis I will check out other layouts latter this evening 
![image](https://github.com/user-attachments/assets/9c40e1c5-76c5-4283-b062-3f4cc69d7f64)

**Comment by Tob3ster97** [comment]
Portrait. Using a ganged Y axis but with Y&A axis limits. Should this still show?

**Comment by Jay-Tech** [comment]
With 4 motors and the Y ganged, no A will not show up.  It might be common for boards to have the 4th axis option listed A or Y1, and Y ganged will be really more like Y0 and Y1 not A.  In the pervious  screen shot I sent that was a 5 motor set X,Y,Y, Z and A axis 
This is what 4 motor with Y ganged looks like 
![image](https://github.com/user-attachments/assets/c6e45963-7e7f-4571-8dfe-437adac7eb96)

**Comment by Tob3ster97** [comment]
Ah okay. Thanks :)
